crates/chain/src/lib.rs

```diff
@@ -77,9 +77,6 @@ pub extern crate serde;
 #[macro_use]
 extern crate std;
 
-/// How many confirmations are needed f or a coinbase output to be spent.
-pub const COINBASE_MATURITY: u32 = 100;
-
```

Comment:
I am unsure if stylistic or for another reason, but BDK does not export types via `pub use` anywhere else in the crate. What do you think @notmandatory @ValuedMammal

Reply:
Although it's a breaking change on the API, it's fixing the API and you can use it from rust-bitcoin (e.g. `bdk_chain::bitcoin::constants::COINBASE_MATURITY`), which is re-exported.

Reply:
I agree this is a breaking change, but it's OK to require downstream projects to switch to explicitly use `bdk_chain::bitcoin::constants::COINBASE_MATURITY` since we'll be rolling this out in the 1.0.0 major release.